picard/ui/mainwindow.py

```diff
@@ -674,7 +679,13 @@ def search(self):
         """Search for album, artist or track on the MusicBrainz website."""
         text = self.search_edit.text()
         type = self.search_combo.itemData(self.search_combo.currentIndex())
-        self.tagger.search(text, type, config.setting["use_adv_search_syntax"])
+        if config.setting["builtin_search"]:
+            if type == "track":
```

Comment:
This means that searching for anything other than a track via the search bar in the top-right corner does nothing at all. The other searches should either fall back on opening the browser and the option indicate this ("when searching for tracks") or be greyed out.